This Commit was to solve posted problems.

Calen934 opened the pull request:
Fizzbuzz was sloved using remainders
String calucuator was solved parsing then turing the strings to ints
	`

patrik-drean commented:
You moved the template files into the _submissions folder. They need to be moved back.

You can make the changes in your branch and push it and it should show updated here.

Calen934 commented:
I over wrote the template files when i did the fist assignment so I renamed the ones I did the work on and deleted the template files that were on my branch
